cmip_model_rmse/create_artifact.jl

```diff
@@ -31,7 +36,28 @@ for i in 1:nvars
         @info "$file_path not found, downloading it (might take a while)"
         rmse_file = Downloads.download(file_urls[i])
         Base.mv(rmse_file, file_path)
-        Base.cp(file_path, joinpath(output_dir, basename(file_path)))
+        output_hdf5 = joinpath(output_dir, basename(file_path))
+        Base.cp(file_path, output_hdf5)
+
+        base_outname = Base.Filesystem.splitext(output_hdf5)[begin]
+
+        # Create CSV file form the HDF5
+        h5 = h5open(file_path, "r") do file
+            for dataset_name in keys(file)
+                @show output_csv = base_outname * "$(dataset_name).csv"
```

Comment:
```suggestion
                @show output_csv = base_outname * "_$(dataset_name).csv"
```

Reply:
(So that the output file name is easier to parse. I don't have a strong preference.)

Reply:
Sounds good.